Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -31,20 +31,26 @@
 NSString *const kSentryLaunchProfileConfigKeyProfilesSampleRate = @"profiles";
 NSString *const kSentryLaunchProfileConfigKeyProfilesSampleRand = @"profiles.sample_rand";
 NSString *const kSentryLaunchProfileConfigKeyContinuousProfiling = @"continuous-profiling";
+NSString *const kSentryLaunchProfileConfigKeyContinuousProfilingV2
+    = @"continuous-profiling-v2-enabled";
+NSString *const kSentryLaunchProfileConfigKeyContinuousProfilingV2Lifecycle
+    = @"continuous-profiling-v2-lifecycle";
 static SentryTracer *_Nullable launchTracer;
 
 #    pragma mark - Private
 
 SentryTracer *_Nullable sentry_launchTracer;
 
 SentryTracerConfiguration *
-sentry_config(NSNumber *profilesRate, NSNumber *profilesRand)
+sentry_configForLaunchProfilerForTrace(
+    NSNumber *profilesRate, NSNumber *profilesRand, SentryProfileOptions *_Nullable profileOptions)
 {
     SentryTracerConfiguration *config = [SentryTracerConfiguration defaultConfiguration];
     config.profilesSamplerDecision =
         [[SentrySamplerDecision alloc] initWithDecision:kSentrySampleDecisionYes
                                           forSampleRate:profilesRate
                                          withSampleRand:profilesRand];
+    config.profileOptions = profileOptions;
```

Comment:
`m`: does this create a new copy of the options? Is it a problem if it is the original object?